lib/opensusebasetest.pm

```diff
@@ -564,6 +564,7 @@ sub remount_tmp_if_ro {
 ## of wait_boot, e.g. see `git grep -l reboot | xargs grep -L wait_boot`
 sub post_fail_hook {
     my ($self) = @_;
+    record_info('post fail hook', 'Post fail hook started for test distribution.');
```

Comment:
Don't we have a box just in before stating "Failed"?

Reply:
Yes, however iirc doesn't happen everywhere, I will just double check before pushing more :)